### ISSUE TITLE
Opening sets ("Unbundle and equip") that are duplicated while the "Stack duplicates" option is on will open all duplicates.

### ISSUE THREAD

**Issue by Drawker** [ValveSoftware/Dota2-Gameplay#11127]
### Description

Opening sets ("Unbundle and equip") that are duplicated (or more) while the "Stack duplicates" option is on will open all duplicates.

I just purchased 23 cache sets and got 2x of most treasures. I went to open the Dawnbreaker with the Unbundle and equip option and it opened both sets, which is really counter intuitive and I can no longer gift the second set to a friend.

**Comment by robinh-valve** [comment]
Could it be that you were unbundling through the All Items tab in the armory and had Only Duplicates option on? When you unbundle the other bundle would just disappears from the grid as it is no longer a duplicate. You should still have the bundle though. We agree that the UX is slightly confusing and will address that in a near future.

**Comment by Drawker** [comment]
Yes, I was in the "All Items" tab but the "Duplicates Only" option was off while the "Stack duplicates" option was on. I just double checked and have no bundles for that set only 2 copies of each individual item.

**Comment by Drawker** [comment]
Quick showcase of the Dawnbreaker set already opened and duplicated as per the bug/feature. In contrast, I disabled the "Stack duplicates" when opening the Alchemist set and I still have 1 bundled set and 1 unbundled.

![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/143716463/9d7ae193-1479-4cb0-9c4f-1be57ddae8ac)
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/143716463/4f961612-4c3a-4275-8ea3-f19a9b8405e8)
